.github/workflows/pull-request.yml

```diff
@@ -26,10 +26,18 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
       - name: Build
-        run: ./gradlew build --scan
+        run: ./gradlew build
       - name: Upload build reports
         if: failure()
         uses: actions/upload-artifact@v3
         with:
           name: build-reports
           path: build/reports/        
+
+
+      - name: Test Report
+        uses: mikepenz/action-junit-report@v4
+        if: success() || failure()
+        with:
+          check_name: 'Integration Tests'
+          report_paths: '**/build/test-results/test/TEST-*.xml'
```

Comment:
Vi måtte slå på en setting i maven for få stacktrace. Tenker at det funker fint i gradle og er kanskje testet med det som var før?

Reply:
jeg fant ikke noe gradle info, så tenkte å sjekke om det funket på denne pr'en og evt lage en feil

Reply:
Sikkert greit å lage en feil her, så slipper man å få det inn i master senere